bin/test

```diff
@@ -72,9 +76,21 @@ function downloadExecutableAndRunTests() {
     getDownloadUrl(function (downloadUrl) {
         console.log('Downloading VS Code into "' + testRunFolderAbsolute + '" from: ' + downloadUrl);
 
-        var stream = remote(downloadUrl, { base: '' })
-            .pipe(vzip.src())
-            .pipe(symdest(testRunFolder));
+        var version = downloadUrl.match(/\d\.\d\.\d/)[0].split('\.');
+        var isTarGz = downloadUrl.match(/linux/) && version[0] >= 1 && version[1] >= 4;
```

Comment:
`downloadUrl` doesn't have an extension: https://vscode-update.azurewebsites.net/1.4.0/linux-x64/stable

I'm not sure how to pull the downloaded file name and make a conditional in the `pipe`?